oteapi/models/sessionupdate.py

```diff
@@ -1,6 +1,41 @@
-"""Pydantic SessionUpdate Data Model."""
-from oteapi.models.genericconfig import AttrDict
+"""Pydantic SessionUpdate Data Model.
 
+Note that the Session Object should be updated according to the
+SessionUpdate. If the Session already has defined a session_type
+and session_id, the code should throw an exception.
+"""
 
-class SessionUpdate(AttrDict):
-    """Session Update Data Model for returning values."""
+from typing import Optional
+from pydantic import ValidationError, root_validator, validator, Field
+from oteapi.models import AttrDict
+
+class OTESession(AttrDict):
```

Comment:
Is this supposed to replace SessionUpdate?

If so, the file should be renamed to otesession.py and all usage should be changed to use SessionUpdate instead. Since, this will be a breaking change, I will suggest that we for backward compatibility keep SessionUpdate a while, but add a deprecation warning.